.github/workflows/prepare-action/action.yml

```
@@ -17,9 +17,13 @@ runs:
       uses: "actions/setup-python@v3"
       id: "setup-python"
       with:
-        python-version: '3.9'
+        python-version: '3.10'
         cache: 'poetry'
```

A suggestion to create a new change to trigger GitHub Actions again.

```suggestion
        python-version: "3.10"
        cache: "poetry"
```